frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelinesTopology.cy.ts

```diff
@@ -680,10 +671,49 @@ describe('Pipeline topology', () => {
       rightDrawer.findRightDrawerDetailsTab().should('be.visible');
       rightDrawer.findRightDrawerLogsTab().should('be.visible');
       rightDrawer.findRightDrawerLogsTab().click();
-      pipelineRunDetails.findLogsSuccessAlert().should('be.visible');
+    };
+
+    beforeEach(() => {
+      initIntercepts();
+    });
+
+    it('test logs paused while loading', () => {
+      cy.interceptK8s(
+        PodModel,
+        mockPipelinePodK8sResource({
+          namespace: projectId,
+          name: 'iris-training-pipeline-v4zp7-2757091352',
+          isPending: true,
+        }),
+      ).as('podLoading');
+      navigateToLogsTab();
+
+      pipelineRunDetails
+        .findLogs()
+        .contains(
+          'sample log for namespace test-project, pod name iris-training-pipeline-v4zp7-2757091352 and for step step-main',
+        );
+      cy.wait('@podLoading');
+
+      cy.interceptK8s(
+        PodModel,
+        mockPipelinePodK8sResource({
+          namespace: projectId,
+          name: 'iris-training-pipeline-v4zp7-2757091352',
+          isPending: false,
+        }),
+      ).as('podTerminated');
+
+      cy.wait('@podTerminated', { timeout: 20000 });
```

Comment:
Is the default timeout not enough?

Reply:
it was having issues, i can remove it and check again.